qiskit_nature/second_q/circuit/library/ansatzes/succd.py

```diff
@@ -77,12 +75,16 @@ def __init__(
                 the occupation of the spin orbitals. As such, the set of generalized excitations is
                 only determined from the number of spin orbitals and independent from the number of
                 particles.
-
+            mirror: boolean flag whether or not to include the symmetrically mirrored double
+                  excitations (refer SUCC_full ansatz), while keeping the original number of circuit
```

Comment:
Nowhere in the code, should there be a reference to "SUCC_full". This has no context and users do not know what this means.
If you want to refer to the old name in same way, I suggest that you add a sentence at the end of this parameter documentation stating something along the lines of this:
> Enabling this parameter will result in the SUCCD ansatz referred to as "q-UCCSD0-full" in reference [1].

And then you add the same reference link as in this class docstring so that `[1]` works.